CHANGES.md

```diff
@@ -0,0 +1,86 @@
+# Changes between Versions
+
+## 2.9.3
+	Release Date: `unreleased`
+
+### NEW FEATURES
+
+- Introduced change log at [CHANGES.md](https://github.com/codice/ddf/blob/master/CHANGES.md)
+
+### API CHANGES
+
+- Updated... 
```

Comment:
Please remove the placeholder text or replace with something like "No Changes".  Since this is for a bug fix release, there should not be any API changes.